ios/ZulipMobile/Info.plist

```diff
@@ -63,5 +63,19 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+  <key>Fabric</key>
+  <dict>
+    <key>APIKey</key>
+    <string>6094380dd00a86a66893866d50ec2f462c09ad72</string>
```

Comment:
Please delete this account and invalidate the key. It should not be posted publicly